diff -aurN imcom-1.30beta9.orig/Makefile.in imcom-1.30beta9/Makefile.in
--- imcom-1.30beta9.orig/Makefile.in	2003-05-21 04:36:37.000000000 +0200
+++ imcom-1.30beta9/Makefile.in	2003-06-22 14:43:16.000000000 +0200
@@ -8,17 +8,17 @@
 	chmod +x imcom CLI.py LogHandler.py AccountCreator.py
 
 install: all
-	install -d -m 0755 ${datadir}/imcom
-	install -d -m 0755 ${datadir}/imcom/modules
-	install -d -m 0755 ${datadir}/imcom/docs
-	install -d -m 0755 ${bindir}
-	install -d -m 0755 ${mandir}/man1/
-	install -m 0755 *.py ${datadir}/imcom
-	install -m 0755 modules/*.py ${datadir}/imcom/modules
-	install -m 0755 docs/*.html ${datadir}/imcom/docs
-	install -m 0755 CONTRIBUTORS LICENSE README README.autostatus TODO WHATSNEW ${datadir}/imcom/docs
-	install -m 0755 imcom ${bindir}
-	install -m 0755 docs/imcom.1 ${mandir}/man1/imcom.1
+	install -d -m 0755 $(DESTDIR)${datadir}/imcom
+	install -d -m 0755 $(DESTDIR)${datadir}/imcom/modules
+	install -d -m 0755 $(DESTDIR)${datadir}/imcom/docs
+	install -d -m 0755 $(DESTDIR)${bindir}
+	install -d -m 0755 $(DESTDIR)${mandir}/man1/
+	install -m 0755 *.py $(DESTDIR)${datadir}/imcom
+	install -m 0755 modules/*.py $(DESTDIR)${datadir}/imcom/modules
+	install -m 0755 docs/*.html $(DESTDIR)${datadir}/imcom/docs
+	install -m 0755 CONTRIBUTORS LICENSE README README.autostatus TODO WHATSNEW $(DESTDIR)${datadir}/imcom/docs
+	install -m 0755 imcom $(DESTDIR)${bindir}
+	install -m 0755 docs/imcom.1 $(DESTDIR)${mandir}/man1/imcom.1
 
 install-docs: install
 
